lib/ansible/config/ansible_builtin_runtime.yml

```diff
@@ -498,12 +498,6 @@ plugin_routing:
       redirect: community.general.ali_instance
     ali_instance_info:
       redirect: community.general.ali_instance_info
-    atomic_container:
-      redirect: community.general.atomic_container
-    atomic_host:
-      redirect: community.general.atomic_host
-    atomic_image:
-      redirect: community.general.atomic_image
```

Comment:
I am not sure why these lines are removed.

Reply:
I'm combining two changes info one. I should arguable make a separate commit/PR.. IIUC the Atomic* development is dead and as such these modules should no longer be used.

Reply:
yes I confirm that.  The atomic CLI is not not maintained anymore (archived on github): https://github.com/projectatomic/atomic